modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -13,101 +16,118 @@ Contact your customer service representative about pricing for this feature.
+All the Mule runtime engines in a cluster share memory, as illustrated below:
 
 image::topology-4-cluster.png[topology_4-cluster]
 
-Mule uses an *active-active* model to cluster Mule runtimes, rather than an *active-passive* model.
+Mule uses an active-active model to cluster Mule runtime engines, rather than an active-passive model.
 
-In an *active-passive* model, one node in a cluster acts as the *primary*, or active node, while the others are *secondary*, or passive nodes. The application in such a model runs on the primary node, and only ever runs on the secondary node if the first one fails. In this model, the processing power of the secondary node(s) is mostly wasted in passive waiting for the primary node to fail.
+In an active-passive model, one node in a cluster acts as the primary, or active node, while the others are secondary, or passive nodes. The application in such a model runs on the primary node and only runs on the secondary node if the first one fails. In this model, the processing power of the secondary node(s) is mostly wasted in passively waiting for the primary node to fail.
 
-In an *active-active* model, no one node in the cluster acts as the primary node; all nodes in the cluster support the application. This application in this model runs on all the nodes, even splitting apart message processing between nodes to expedite processing across nodes.
+In an active-active model, no one node in the cluster acts as the primary node; all nodes in the cluster support the application. The application in this model runs on all the nodes and splits apart message processing between nodes to expedite processing.
 
-== About Queues
+== Queues
 
-You can set up a *VM queue* explicitly to *load balance* across Mule runtimes (nodes). Thus, if your entire application flow contains a sequence of child flows, the cluster can assign each successive child flow to whichever Mule runtime (node) happens to be available at the time. Potentially, the cluster can process a single message on multiple nodes as it passes through the VM endpoints in the application flow, as illustrated below:
+You can set up a VM queue explicitly to load balance across Mule runtime engines (nodes). Thus, if your entire application flow contains a sequence of child flows, the cluster can assign each successive child flow to whichever Mule runtime engine (node) happens to be available at the time. Potentially, the cluster can process a single message on multiple nodes as it passes through the VM endpoints in the application flow, as illustrated below:
 
 image::load-balancing.png[load_balancing]
 
-== About High-Reliability Applications
+== High-Reliability Applications
 
-A *high-reliability* application must meet the following requirements:
+A high-reliability application must meet the following requirements:
 
 . Zero tolerance for message loss
 . A reliable underlying enterprise service bus (Mule)
```

Comment:
The concept of Mule as an ESB is quite old, how can we rewrite this?

Reply:
That line could be replace with the definition of Mule we have in the runtime overview:
* A reliable integration engine (Mule)